format-registry/initdata/index-respec.html

```diff
@@ -153,19 +153,19 @@
             <td>
               cenc
             </td>
-            <td><a href="cenc.html">ISO Common Encryption Protection System Specific Header (PSSH) Boxes Initialization Data Format</a></td>
+            <td>[[!EME-INITDATA-CENC]]</td>
```

Comment:
I ended up using both, so that you get the direct link for usability and the proper reference in the reference section of the spec.